### PR TITLE
get-started-UI changes

### PR DESCRIPTION
**Pull request by suvarnakale**
get started UI changes- margin padding changes and checkbox change.

### CODE REVIEW

File: src/app/admin/get-started/get-started.component.scss [hunk 1]
```diff
@@ -70,7 +72,7 @@ a:focus {
 }
 
 .navbar {
-  padding: 15px 10px;
+  padding: px 10px;
```
Comment (Pratikshakhandagale) [inline src/app/admin/get-started/get-started.component.scss:75]: @suvarnakale - only px? if not required then set 0 only

File: src/app/admin/get-started/get-started.component.html [hunk 2]
```diff
@@ -115,9 +117,10 @@
                   class="form-check-input"
                   type="checkbox"
                   value=""
-                  id="flexCheckDefault"
+                  id="flexCheckDefault3"
```
Comment (Pratikshakhandagale) [inline src/app/admin/get-started/get-started.component.html:120]: @suvarnakale can you set relavent name for this check box like- ownershipCheckbox like this. Check for all card checboxes

File: src/app/admin/get-started/get-started.component.scss (hunk 1 above)
Comment (suvarnakale) [reply src/app/admin/get-started/get-started.component.scss:75]: okay mam

File: src/app/admin/get-started/get-started.component.scss [hunk 3]
```diff
@@ -55,10 +57,10 @@ p {
   background-color: none;
 }
 .top-padding-title {
-  padding-top: 15px;
+  adding-top: 0px;
```
Comment (Pratikshakhandagale) [inline src/app/admin/get-started/get-started.component.scss:60]: @suvarnakale correct spelling - padding

**Pull request merged**